tests/instance_pools/test_cli.py

```diff
@@ -0,0 +1,118 @@
+# Databricks CLI
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint:disable=redefined-outer-name
+
+import json
+import mock
+import pytest
+from tabulate import tabulate
+from click.testing import CliRunner
+
+import databricks_cli.instance_pools.cli as cli
+from databricks_cli.utils import pretty_format
+from tests.utils import provide_conf
+
+CREATE_RETURN = {'pool_id': 'instance_pool_id'}
+CREATE_JSON = '{"name": "instance_pool_name"}'
+EDIT_JSON = '{"pool_id": "instance_pool_id"}'
```

Comment:
I meant the other way...the official field names are `instance_pool_id` and `instance_pool_name`. So I was thinking, to eliminate confusion:

```
CREATE_RETURN = {'instance_pool_id': 'test_id'}
CREATE_JSON = '{"instance_pool_name": "test_name"}'
EDIT_JSON = '{"instance_pool_id": "test_id"}'
```